src/TitleDetails/utils.js

```diff
@@ -0,0 +1,61 @@
+import {
+  find,
+  some,
+} from 'lodash';
+
+export const checkInItems = (pieces, mutator) => {
+  const linesMap = {};
+
+  pieces.forEach(piece => {
```

Comment:
mb reduce is better here

Reply:
it could be discussed. it was moved from `ui-orders`